src/wp-includes/block-bindings/class-wp-block-bindings.php

```diff
@@ -0,0 +1,281 @@
+<?php
```

Comment:
The name of the file needs to align with the class name:

`src/wp-includes/block-bindings/class-wp-block-bindings-registry.php`